File: modulo1/segmento3/examen/Ejercicio2.java
import java.util.Scanner;

public class Ejercicio2{
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);

		System.out.println("Ingrese numero de inicio: ");
		Integer numini = Integer.parseInt(teclado.nextLine());
	
		System.out.println("Ingrese numero de fin: ");
		Integer numfin = Integer.parseInt(teclado.nextLine());
	
		System.out.println("Ingrese numero de referencia: ");
		Integer numref = Integer.parseInt(teclado.nextLine());
		
		Integer resultado=0;

		if (numfin>numini){
			if(numfin>0 && numini>0 && numref>0){
				for ( Integer i=numini; i<=numfin; i++){
					resultado=i%numref;
					if(resultado==0){
						System.out.println(i);
					}				
				}
			}else{
				System.out.println("Los numeros ingresados deben ser mayor a 0");

			}
		}else{
			System.out.println("El numero de fin debe ser mayor que el numero de inicio");
			
		}	
	}


}

File: modulo1/segmento3/apunte7Funciones/ejercicio2.py
def multiplos(num1,num2):
	respuesta=""
	if num1%num2==0:
		respuesta= str(num1) + " es multiplo de " + str(num2)
	elif num2%num1==0:
		respuesta= str(num2) + " es multiplo de " + str(num1)
	else :
		respuesta="Los numeros ingresados no son multiplos"
	
	return respuesta

num1=int(input("Ingrese un numero: " ))
num2=int(input("Ingrese un numero: " ))

print(multiplos(num1,num2))
		
